package.json

```diff
@@ -50,6 +50,8 @@
     "can-define": "^1.3.3",
     "can-list": "^3.2.0",
     "can-map": "^3.3.1",
+    "can-stache": "^3.6.0",
```

Comment:
Can this be tested without depending on can-stache and can-stache-bindings? This will add a cyclic dependency that will make it tough to release things in the future.